study.md

```diff
@@ -246,5 +267,9 @@ Pick a few design heuristics. Describe some common, named refactorings you can
 use approach a good design.
 
 ```md
-<!-- your answer here -->
+I think for me, keeping methods to 5 lines or shorter is a key heuristic.  I think
+I can sometimes get carried away with methods or functions that swell to much larger.
+Even though this rule can be broken, I feel this is a good one, and is something
+I am going to work on moving forward. 
+
```

Comment:
Great start. What would you use to DRY up duplicated logic? Maybe "Extract Method". Have a look at some named refactorings, like "Extract Method", to level up.